MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -277,6 +345,13 @@ void EnggDiffractionPresenter::processFocusTexture() {
 
   } else if (focusMode == 1) {
     g_log.debug() << " focus mode selected Focus Sum Of Files " << std::endl;
+    g_sumOfFilesFocus = "texture";
```

Comment:
This is actually from a different PR. Could you handle these basic/cropped/texture with an enum variable instead of a string?